pkg/sloop/queries/queryd3heatmap_test.go

```diff
@@ -106,6 +106,7 @@ func helper_UrlValues() url.Values {
 	return map[string][]string{
 		NamespaceParam: []string{AllNamespaces},
 		KindParam:      []string{AllKinds},
+		BucketingParam: {"1m"},
```

Comment:
should we put this 1m as a constant somewhere, so it can be used/updated eaisier?